.github/workflows/build_aws_scheduled.yml

```diff
@@ -0,0 +1,36 @@
+name: AWS AMI scheduled publish
+
+on:
+  schedule:
+    # Every Sunday at 23:00 UTC
+    - cron: "00 23 * * 0"
```

Comment:
fun fact, this was `23 00` in AWS, I'm pretty sure it meant to be `00 23` 😅

<img width="178" alt="image" src="https://user-images.githubusercontent.com/19969687/203090343-122dc4c7-7eae-4faf-9bf1-d4b53f6f480e.png">